### PR TITLE
[UXE-2717] feat: import static and connect with github form page

### PR DESCRIPTION
**Pull request by peterpaulo-azion**
### What is the new behavior introduced by this PR?

* flow: connect/oauth with github
* vulcan presets and mode services (with tests)
* added create variables form section
* script runner execution api route 

### Does this PR introduce breaking changes?
- [x] No
- [ ] Yes 

### Does this PR introduce UI changes? Add a video or screenshots here.

https://github.com/aziontech/azion-console-kit/assets/109550332/33cde018-7120-43f9-ba31-4024ec04a487

### Does it have a link on Figma?

https://www.figma.com/file/GvpnDYOyV6KnK6ib0jYJVw/Dev-Journey?type=design&node-id=390-12269&mode=design&t=N9h7VblzxDQYSJaj-0

<hr />

#### Make sure your pull request fits the checklist below (when applicable):

- [x] The issue title follows the format: [ISSUE_CODE] TYPE: TITLE
- [x] Commits are tagged with the right word (fix, feat, test, etc)
- [x] User inputs are sanitized/protected against malicious attacks
- [x] Tags are added to the PR

#### These changes were tested on the following browsers:
- [x] Chrome
- [ ] Edge
- [ ] Firefox
- [ ] Safari

### CODE REVIEW

File: src/services/script-runner-service/create-script-runner-execution-service.js [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+    }
+  ]
+}
+
+const adapt = (payload) => {
+  const parsedVariables = parseVariables(payload.newVariables)
+  const parsedFormData = parseFormData(payload)
+
+  return {
+    image: 'azionedge/runner-static-website-with-cli-stage:2.1-alpine3.16',
```
Comment (matheus-sunderhus) [inline src/services/script-runner-service/create-script-runner-execution-service.js:72]: colcoar isso em uma variável de ambiente, pois essa imagem está usando sendo o de stage.
Como não é um segredo , isso pode ser adicionado via Vite Env .

Comment (peterpaulo-azion) [reply src/services/script-runner-service/create-script-runner-execution-service.js:72]: bem observado, acabou passando em meio aos commits da feat.

Comment (lucasmendes21) [reply src/services/script-runner-service/create-script-runner-execution-service.js:72]: resolvendo esse ponto por mim ta de boas

**Review by matheus-sunderhus — DISMISSED** [review]

**Review by lucasmendes21 — DISMISSED** [review]

File: src/services/script-runner-service/create-script-runner-execution-service.js (hunk 1 above)
Comment (peterpaulo-azion) [reply src/services/script-runner-service/create-script-runner-execution-service.js]: Done!

**Review by matheus-sunderhus — APPROVED** [review]

**Review by lucasmendes21 — APPROVED** [review]

**Pull request merged**